data/assets/scene/solarsystem/telescopes/jwst/trail.asset

```diff
@@ -76,18 +74,19 @@ local JWSTTrailCruise = {
 -- This does NOT co-rotate with L2 and gives a saddle look of the trail
 local JWSTTrailOrbit = {
   Identifier = "JWSTTrailOrbit",
-  Parent = transforms.L2Position.Identifier,
+  Parent = L2transforms.L2Position.Identifier,
   TimeFrame = {
     Type = "TimeFrameInterval",
-    Start = L2orbitInsertionTime,
+    Start = transforms.L2orbitInsertionTime,
     End = EndTime
   },
   Renderable = {
     Type = "RenderableTrailOrbit",
+    Enabled = asset.enabled,
     Translation = {
       Type = "SpiceTranslation",
       Target = kernels.ID.JWST,
-      Observer = transforms.ID.L2,
+      Observer = L2transforms.ID.L2,
```

Comment:
I get an error here when trying to run the JWST profile. I think this is supposed to be `L2Kernels.ID.L2` instead.